CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [3.x]
+### Added
+- Add the `AbstractAdmin::showMosaicButton` method from @tim96
```

Comment:
We does not. Let keeps it clear. GitHub already have nice stuff to show which ones worked on this project.